src/download.jl

```diff
@@ -91,7 +91,7 @@ from the rules of the HTTP.
 """
 function download(url::AbstractString, local_path=nothing, headers=Header[]; update_period=1, kw...)
     format_progress(x) = round(x, digits=4)
-    format_bytes(x) = !isfinite(x) ? "∞ B" : Base.format_bytes(x)
+    format_bytes(x) = !isfinite(x) ? "∞ B" : Base.format_bytes(round(x))
```

Comment:
```suggestion
    format_bytes(x) = !isfinite(x) ? "∞ B" : Base.format_bytes(round(Int, x))
```

Maybe let's actually return an `Int`
?